accountStorageExporter/vars.go

```diff
@@ -0,0 +1,28 @@
+package main
```

Comment:
there is a lot of boiler plate in this app: go/mof/go.sum, vars handling, the simplePathManagerComponent, and so on. We can refactor these by actually having only one project with multiple binaries. In this way we can share the common components

Reply:
done, restructured the entire repository